make/upgrade

```diff
@@ -21,12 +21,25 @@ stampy "${GIT_ROOT}/uaa_metrics.csv" "$0" make-run::upgrade start
 
 DOMAIN=${DOMAIN:-cf-dev.io}
 
-helm upgrade ${RELEASE} helm \
-     --namespace "${NAMESPACE}" \
-     --values "../../bin/settings.yaml" \
-     --set "env.DOMAIN=${DOMAIN}" \
-     --set "kube.external_ips[0]=$(getent hosts "${DOMAIN}" | awk 'NR=1{print $1}')" \
-     "$@"
+helm_args=(
+    --namespace "${NAMESPACE}"
+    --values "../../bin/settings.yaml"
+    --set "env.DOMAIN=${DOMAIN}"
+    --set "kube.storage_class.persistent=${STORAGE_CLASS}"
```

Comment:
Do we need to set this?

Reply:
I thought it is appropriate to set it since we are not using `--reuse-values` (and I don't think we want it?).

Reply:
I thought we _didn't_ want it to ensure the default storage class stuff worked (since we'd just never set it; so `--reuse-values` never comes into play).